package.json

```diff
@@ -32,6 +32,7 @@
     "db-migrate": "^0.11.11",
     "db-migrate-pg": "^1.2.1",
     "dotenv-load": "^2.0.0",
+    "email-validator": "^2.0.4",
```

Comment:
Does this library not validate for us then? If so why do we need it?

Reply:
We no longer need it, forgot to remove! 😅